test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,82 @@
+const axios = require("axios");
+const { expect } = require("chai");
+const { StatusCodes } = require("http-status-codes");
+
+const url = 'https://httpbin.org/ip';
+
+describe("First Api Tests", () => {
+    it("Consume GET Service", async () => {
+        const response = await axios.get(url);
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property("origin");
+    });
+
+
+    it("Consume GET Service with query parameters", async () => {
+        const query = {
+            name: "John",
+            age: "31",
+            city: "New York",
+        };
+
+        const response = await axios.get("https://httpbin.org/get", { query });
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+
+
+    it("Consume HEAD Service", async () => {
+        const response = await axios.head("https://httpbin.org/headers");
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.headers).to.have.property("server");
+    });
+
+
+    it("Consume PUT Service with query parameters", async () => {
+        const queryPut = {
```

Comment:
in this case, as we are sending this as body, instead of query, you can rename the variable to be called as `body`, or `payload` or `content`, will be more clear.